src/pages/msfs/hardware/thrustmaster-tfrp.mdx

```diff
@@ -0,0 +1,36 @@
+import { Callout } from 'nextra/components';
+import { Steps } from 'nextra-theme-docs';
+
+# 
+
+[Thrustmaster Flight Rudder Pedals](https://www.thrustmaster.com/fr-fr/products/t-flight-rudder-pedals) is a popular hardware pedals unit for many flight simulators.
+It works great with a Honeycomb Alpha Bravo combo for much cheaper than the Honeycomb charlie gear.
+
+To make it recognizable by Linux one tweak is needed.
+
+## Setup
+
+<Callout type="info" emoji="ℹ️">
+  **Difficulty:** Medium
+</Callout>
+
+<Steps>
+
+#### Create a udev rule
```

Comment:
I think this is supposed to be `###`, unless I am mistaken about markdown in Nextra.